build.gradle

```diff
@@ -28,6 +28,14 @@ ext.ext_classpath = ['com.google.code.gson:gson:2.2.4',
   'io.dropwizard.metrics:metrics-core:3.1.2']
 ext.target_dir = "$distsDir"
 ext.target_java8_dir = "${ext.target_dir}/java8"
+ext.target_javadoc_dir = "${ext.target_dir}/docs/javadoc"
+
+if (JavaVersion.current().isJava8Compatible()) {
+  tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Hi @dlaboss Thank you for review!
It looks only few warning in javadoc. I removed Xdoclint:none option.
I will create another jira issue for remove these warnings

And, except for a few differences, the results of Javadoc of ant and gradle are almost matches.(compare with ```diff -r build/distributions/docs/javadoc/ target/docs/javadoc/```)
The only thing I worry about is we could not see the JavaDoc that inherited from interface
(e.g ```diff -r build/distributions/docs/javadoc/quarks/connectors/file/FileWriterPolicy.html target/docs/javadoc/quarks/connectors/file/FileWriterPolicy.html``` 
And the ant output added description from interface like ```Description copied from interface: quarks.connectors.file.runtime.IFileWriterPolicy``` but gradle is not)

Unfortunately, I still have not found a way to solve this problem. I am still looking for a solution, but could you tell me if you know the related javadoc options?